src/actions/index.js

```diff
@@ -9,10 +9,13 @@ export const DECREMENT = 'DECREMENT';
 // logic of updating the central Redux store. That
 // is left to the reducer(s).
 
+// console.log(props);
+console.log(this.props);
+
 export const increment = () => {
-  // Fill in this function
+  return {type: INCREMENT}
```

Comment:
Very good! Great job using the variables instead of strings here. Since we have no payload to send, we don't need it in this return object, good work identifying that.